.gitignore

```diff
@@ -21,3 +21,6 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+*.iml
```

Comment:
Can you add some comments, like what kind of files are these?

Reply:
sure, i have added the iml and .idea files

Reply:
i have added details in the PR description.

Reply:
Thanks, I would have seen that... my bad!